reference/configuration/framework.rst

```diff
@@ -3092,7 +3092,7 @@ A list of workflows to be created by the framework extension:
         // config/packages/workflow.php
         $container->loadFromExtension('framework', [
             'workflows' => [
-                'my_workflow' => // ...
+                'my_workflow' => [ /* ... */ ]
```

Comment:
I'm not sure I like this (and other `...` related changes in this PR). It's more complex, so harder to understand (while it just means "there is some value here already"). Moreover, it "hides" the fact that more is needed here when you directly copy-past the code. A PHP error is a good indicator that you need some more info here.

Reply:
I dont like this one too much either =)

But `my_workflow` MUST be an array, the other code examples are showing that, but not PHP.